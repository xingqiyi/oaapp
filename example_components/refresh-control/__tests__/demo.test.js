import { webDemoTest } from '../../../tests/shared/demoTest';

webDemoTest('refresh-control');
