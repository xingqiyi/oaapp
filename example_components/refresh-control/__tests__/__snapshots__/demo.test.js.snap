// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ./components/refresh-control/demo/basic.md correctly 1`] = `
<div
  class="am-list-view-scrollview"
  style="position:relative;overflow:hidden;flex:1;height:0;border:1px solid #ddd;margin:0.1rem 0;"
>
  <div
    class="am-list-view-scrollview-content am-list"
    style="position:absolute;min-width:100%;"
  >
    <div
      class="am-refresh-control-ptr"
    >
      <div
        class="am-refresh-control-ptr-icon"
      >
        <div
          class="am-refresh-control-pull"
        >
          <span>
            下拉可以刷新
          </span>
        </div>
        <div
          class="am-refresh-control-release"
        >
          <span>
            松开立即刷新
          </span>
        </div>
      </div>
      <div
        class="am-refresh-control-ptr-loading"
      >
        <svg
          class="am-icon am-icon-loading am-icon-md"
        >
          <use
            xlink:href="#loading"
          />
        </svg>
      </div>
    </div>
    <div
      class="am-list-body"
    >
      <div
        class="list-view-section-body"
      >
        <div
          style="padding:0.08rem 0.16rem;background-color:white;"
        >
          <h3
            style="padding:2px;margin-bottom:0.08rem;border-bottom:1px solid #F6F6F6;"
          >
            Eat the week
          </h3>
          <div
            style="display:flex;"
          >
            <img
              alt="icon"
              src="https://zos.alipayobjects.com/rmsportal/hfVtzEhPzTUewPm.png"
              style="height:1.28rem;margin-right:0.08rem;"
            />
            <div
              style="display:inline-block;"
            >
              <div
                style="margin:0.1rem 0 0.2rem 0;"
              >
                不是所有的兼职汪都需要风吹日晒-r0
              </div>
              <div>
                <span
                  style="font-size:1.6em;color:#FF6E27;"
                >
                  35
                </span>
                元/任务
              </div>
            </div>
          </div>
        </div>
        <div
          style="background-color:#F5F5F9;height:8px;border-top:1px solid #ECECED;border-bottom:1px solid #ECECED;"
        />
        <div
          style="padding:0.08rem 0.16rem;background-color:white;"
        >
          <h3
            style="padding:2px;margin-bottom:0.08rem;border-bottom:1px solid #F6F6F6;"
          >
            McDonald's invites you
          </h3>
          <div
            style="display:flex;"
          >
            <img
              alt="icon"
              src="https://zos.alipayobjects.com/rmsportal/XmwCzSeJiqpkuMB.png"
              style="height:1.28rem;margin-right:0.08rem;"
            />
            <div
              style="display:inline-block;"
            >
              <div
                style="margin:0.1rem 0 0.2rem 0;"
              >
                不是所有的兼职汪都需要风吹日晒-r1
              </div>
              <div>
                <span
                  style="font-size:1.6em;color:#FF6E27;"
                >
                  35
                </span>
                元/任务
              </div>
            </div>
          </div>
        </div>
        <div
          style="background-color:#F5F5F9;height:8px;border-top:1px solid #ECECED;border-bottom:1px solid #ECECED;"
        />
        <div
          style="padding:0.08rem 0.16rem;background-color:white;"
        >
          <h3
            style="padding:2px;margin-bottom:0.08rem;border-bottom:1px solid #F6F6F6;"
          >
            Meet hotel
          </h3>
          <div
            style="display:flex;"
          >
            <img
              alt="icon"
              src="https://zos.alipayobjects.com/rmsportal/dKbkpPXKfvZzWCM.png"
              style="height:1.28rem;margin-right:0.08rem;"
            />
            <div
              style="display:inline-block;"
            >
              <div
                style="margin:0.1rem 0 0.2rem 0;"
              >
                不是所有的兼职汪都需要风吹日晒-r2
              </div>
              <div>
                <span
                  style="font-size:1.6em;color:#FF6E27;"
                >
                  35
                </span>
                元/任务
              </div>
            </div>
          </div>
        </div>
        <div
          style="background-color:#F5F5F9;height:8px;border-top:1px solid #ECECED;border-bottom:1px solid #ECECED;"
        />
        <div
          style="padding:0.08rem 0.16rem;background-color:white;"
        >
          <h3
            style="padding:2px;margin-bottom:0.08rem;border-bottom:1px solid #F6F6F6;"
          >
            Eat the week
          </h3>
          <div
            style="display:flex;"
          >
            <img
              alt="icon"
              src="https://zos.alipayobjects.com/rmsportal/hfVtzEhPzTUewPm.png"
              style="height:1.28rem;margin-right:0.08rem;"
            />
            <div
              style="display:inline-block;"
            >
              <div
                style="margin:0.1rem 0 0.2rem 0;"
              >
                不是所有的兼职汪都需要风吹日晒-r3
              </div>
              <div>
                <span
                  style="font-size:1.6em;color:#FF6E27;"
                >
                  35
                </span>
                元/任务
              </div>
            </div>
          </div>
        </div>
        <div
          style="background-color:#F5F5F9;height:8px;border-top:1px solid #ECECED;border-bottom:1px solid #ECECED;"
        />
        <div
          style="padding:0.08rem 0.16rem;background-color:white;"
        >
          <h3
            style="padding:2px;margin-bottom:0.08rem;border-bottom:1px solid #F6F6F6;"
          >
            McDonald's invites you
          </h3>
          <div
            style="display:flex;"
          >
            <img
              alt="icon"
              src="https://zos.alipayobjects.com/rmsportal/XmwCzSeJiqpkuMB.png"
              style="height:1.28rem;margin-right:0.08rem;"
            />
            <div
              style="display:inline-block;"
            >
              <div
                style="margin:0.1rem 0 0.2rem 0;"
              >
                不是所有的兼职汪都需要风吹日晒-r4
              </div>
              <div>
                <span
                  style="font-size:1.6em;color:#FF6E27;"
                >
                  35
                </span>
                元/任务
              </div>
            </div>
          </div>
        </div>
        <div
          style="background-color:#F5F5F9;height:8px;border-top:1px solid #ECECED;border-bottom:1px solid #ECECED;"
        />
      </div>
    </div>
  </div>
</div>
`;
